Classes/Controller/AbstractFluxController.php

```diff
@@ -24,7 +24,7 @@
 use TYPO3\CMS\Extbase\Mvc\Controller\ActionController;
 use TYPO3\CMS\Extbase\Mvc\Exception\StopActionException;
 use TYPO3\CMS\Extbase\Mvc\View\ViewInterface;
-use TYPO3\CMS\Extbase\Mvc\Response;
+use TYPO3\CMS\Extbase\Mvc\response;
```

Comment:
Unintended change? The class name, if present, definitely starts with an uppercase "R" and this would cause problems with composer autoloading on TYPO3 versions that have this class.

Reply:
Has been restored.